src/herbivoresAndCarnivores.js

```diff
@@ -1,17 +1,44 @@
 'use strict';
-
 class Animal {
-  // write your code here
+  constructor(name, health = Animal.defaultHealth) {
+    this.name = name;
```

Comment:
You don't need health as a constructor argument, as it's always 100. Move it as 
`this.health = 100;` to the constructor body